gbfs.md

```diff
@@ -368,6 +378,85 @@ alerts            | Yes         | Array - alert objects each indicating a separa
 
+### geofencing_zones.json
+This file is optional. All fields indicated as “Required” are only required if geofencing_zones.json is supplied.
+
+Field Name                           | Required    | Defines
+-----------------------------------  | ------------| ----------
+geofencing_zones{}                   | Yes         | A GeoJSON FeatureCollection (as described by the IETF [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.3)). Each geofenced zone and its associated rules and attributes is described as an object within the array of features, as follows.
+\- type                              | Yes         | String- “Feature” (as per IETF [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.3)).
+\- geometry                          | Yes         | A GeoJSON Multipolygon that describes where rides might not be able to start, end, go through, or have other limitations. A clockwise arrangement of points defines the area enclosed by the polygon, while a counterclockwise order defines the area outside the polygon [(right-hand rule)](https://tools.ietf.org/html/rfc7946#section-3.1.6). All geofencing zones contained in this list are public.
+\- properties{}                      | Yes		   | As defined below, describes travel allowances and limitations.
+&emsp;- name                         | Optional    | String- Name of the geofencing zone.
+&emsp;- rules[]                      | Optional    | Array that contains one object per rule as defined below. In the event of overlapping or colliding rule, the earlier defined rule (in order of the JSON file) takes precedence.
+&emsp;&emsp;- vehicle_type_id        | Optional    | Array of IDs of vehicle types for which any restrictions should be applied (see vehicle type definitions in [PR #136](https://github.com/NABSA/gbfs/pull/136)). If vehicle_type_ids are not specified, then restrictions apply to all vehicle types.
+&emsp;&emsp;- ride_start_allowed     | Yes		   | 1/0 Boolean- Is a ride allowed to start in this zone? <br /> <ul><li>1 - Undocked (“free bike”) ride can start in this zone.</li> <li>0 - Undocked (“free bike”) ride cannot start in this zone.</li> </ul>
+&emsp;&emsp;- ride_end_allowed       | Yes		   | 1/0 Boolean- Is a ride allowed to end in this zone? <br /> <ul><li>1 - Undocked (“free bike”) drop-offs are allowed in this zone.</li> <li>0 - Undocked (“free bike”) drop-offs are not allowed in this zone.</li> </ul>
+&emsp;&emsp;- ride_through_allowed   | Yes		   | 1/0 Boolean- Is a ride allowed to travel through this zone? <br /> <ul><li>1 - Ride can travel through this zone.</li> <li>0 - Ride cannot travel through this zone.</li></ul>
+
+Example:
+```json
+{
+   "last_updated":1434054678,
+   "ttl":0,
+   "version":"2.X",
+   "data":{
+      "geofencing_zones":{
+         "type":"FeatureCollection",
+         "features":[
+            {
+               "type":"Feature",
+               "properties":{
+                  "name":"Downtown Waterfront",
```

Comment:
I would recommend three optional additional keys next to name here:

 * `start_date`: string ([ISO 8601](https://de.wikipedia.org/wiki/ISO_8601)) | Date and time of the start of the geo-fencing zone
 * `end_date`: string ([ISO 8601](https://de.wikipedia.org/wiki/ISO_8601)) | Date and time of the end of the geo-fencing zone
 * `description`: string | Description about the geo-fencing zone

These could be used to display temporary no-park zones for which cities place, for example for Christmas markets or street construction sites.

Reply:
Hi @Empty2k12: If `start_date` and `end_date` were included would you or another application developer commit to implementing that in software?

Reply:
Yes, we would love to implement `start_date` for our unofficial GBFS feeds we're providing.

We'd also like to support `start_date` and `end_date` for the GBFS explorer application we're actively developing. For that we'd implement these changes into our GraphQL wrapper for GBFS ([GbfsQL](https://github.com/mapintelligenceagency/gbfsQL)) and incorporate displaying that into the viewer frontend.